src/linter.js

```diff
@@ -223,9 +223,8 @@ export default class Linter {
           log.info('Retrieving metadata from manifest.json');
           return this.io.getFileAsString(MANIFEST_JSON)
             .then((json) => {
-              var result = new ManifestJSONParser(
-                json, this.collector).getMetadata();
-              return result.metadata;
+              var manifestParser = new ManifestJSONParser(json, this.collector);
```

Comment:
Whatever works. I've been trying to always use let unless var is needed. My rationale is that a code refactor might introduce a new block (a line down from your closure, for example) that could leak the variable.